invisible_cities/io/channel_param_io_test.py

```diff
@@ -35,14 +37,46 @@ def test_generic_parameters(config_tmpdir):
     val_list = np.concatenate(val_list)
     with tb.open_file(filename) as dataIn:
 
-        tblRead = dataIn.root.FITPARAMS.FIT_test_generic
-        assert tblRead.nrows == n_rows
-        assert len(tblRead.colnames) == len(generic_params) + 1
-        assert tblRead.colnames == col_list
-        all_values = [ list(x)[1:] for x in tblRead[:] ]
+        tbl_names, param_names, tbls = basic_param_reader(dataIn)
+        assert len(tbls) == 1
+        assert tbls[0].nrows == n_rows
+        assert len(param_names[0]) == len(col_list)
+        assert param_names[0] == col_list
+        all_values = [ list(x)[1:] for x in tbls[0][:] ]
         assert_allclose(np.concatenate(all_values), val_list)
 
 
+def test_generator_param_reader(config_tmpdir):
+    filename = os.path.join(config_tmpdir, 'test_param.h5')
+
+    outDict = {}
+    val_list = []
+    n_rows = 5
+    with tb.open_file(filename, 'w') as dataOut:
+        pWrite = channel_param_writer(dataOut, sensor_type="test",
```

Comment:
Minor point: our style guide agrees with PEP8 that it should be `data_out` rather than `dataOut` and similarly for `dataIn`, `outDict` etc.

Reply:
easily fixed...